apps/web/components/Map/Map.tsx

```
@@ -0,0 +1,88 @@
+interface Props {
+  boardUuid: string;
+  userId: string;
+  ref: React.RefObject<NaverMap | null>;
+}
+
+interface ClientRect {
+  userId: string;
+  rect: NaverRectangle;
+}
+
+export function Map({boardUuid, userId, ref: mapRef}: Props) {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)